### PR TITLE
Added survey logic for randomized experimental round

### PR DESCRIPTION
**Pull request by phoebexy**

### CODE REVIEW

File: front/src/containers/Batch.js [hunk 1]
```diff
@@ -929,7 +929,8 @@ class Batch extends React.Component {
     let numTask = batch.currentRound - 1; // standard flow
     if (
       batch.teamFormat === "single" &&
-      batch.bestRoundFunction !== "do not reconvene"
+      batch.bestRoundFunction !== "do not reconvene" && // TODO Status
```
Comment (markwhiting) [inline front/src/containers/Batch.js]: What's the todo?

File: server/controllers/admin.ts [hunk 2]
```diff
@@ -187,7 +187,7 @@ export const addBatch = async function(req, res) {
         // expRound is randomized. It's [numRounds] or [numRounds - 1]
         const min = newBatch.numRounds - 1;
         const roundNumber = Math.floor(Math.random() + 0.5) + min; //random int from min to max
-
+        console.log("Random Experiment Round: " + roundNumber);
```
Comment (markwhiting) [inline server/controllers/admin.ts]: Do we need to log this? Isn't it stored in the DB? (I don't mind either way, I'm just always conscious that putting more in the logs usually makes it harder to find what we want to see there)

**Review by markwhiting — COMMENTED** [review]
I added a few comments. Overall looks fine. This fixes it locally?

File: server/controllers/admin.ts (hunk 2 above)
Comment (phoebexy) [reply server/controllers/admin.ts]: I'll remove!

File: front/src/containers/Batch.js (hunk 1 above)
Comment (phoebexy) [reply front/src/containers/Batch.js]: Oops, forgot to remove.

**Pull request merged**